modules/face/samples/sampleKazemi.cpp

```diff
@@ -0,0 +1,113 @@
+#include <iostream>
+#include <vector>
+#include <fstream>
+#include "opencv2/face/face_alignment.hpp"
```

Comment:
Is it necessary to include partial headers of the face module? To me it seems that we would prefer `#include "opencv2/face/face.hpp"`. That should include the alignment part internally. Its to keep stuff simple for users.